extensions/skyportal/static/js/components/Group.jsx

```diff
@@ -0,0 +1,568 @@
+import React, {useEffect, useState} from 'react';
```

Comment:
@dmitryduev could these extensions/duplications be obviated if we simply added a few features/changes on the SP side?

Reply:
that's the hope :) I'm happy to submit the corresponding PRs on the SP side once we converge on what could be "transferred"